bin/compile

```diff
@@ -130,7 +133,14 @@ fi
 
 # Moving to the build, after running the compile. All will be deleted and removed to app directory.
 # Copying the result, the files will be kept in storage.
-cp -R $PUB_CACHE $BUILD_DIR
+cp -R $PUB_CACHE $APP_DIR
+
+if [ -n "$FLUTTER_DEPLOY_DIR" ]; then
+  cp -r "$APP_DIR" "$BUILD_DIR/$FLUTTER_DEPLOY_DIR"
+fi
+
+# To see files size - we need to remove some flutter related builds to meet compression slug norms
+du -h -d 2 | sort -n
```

Comment:
To nadal jest potrzebne?

Reply:
zabrakło w tym PR jednej najważniejszej rzeczy - to nadal nie działa. nie udało mi się wczoraj szybciej niż w ciagu 1h zmniejszyć rozmiaru skompiloawnego ślimaka ;( zgłosiłem ten PR raczej żebyś mogł na nim działać, dlatego też zostawiłem to, bo dobry baromoter